metadata.json

```diff
@@ -37,14 +36,13 @@
     {
       "operatingsystem": "OracleLinux",
       "operatingsystemrelease": [
-        "6",
         "7"
       ]
     },
     {
       "operatingsystem": "Scientific",
```

Comment:
This block is either missing Scientific 7 or can be removed altogether since we are no longer supporting Scientific OSs for this module.